internal/c/libqb.cpp

```diff
@@ -14684,6 +14685,14 @@ void qbg_sub_view(int32 x1, int32 y1, int32 x2, int32 y2, int32 fillcolor, int32
 }
 
 void qbg_sub_locate(int32 row, int32 column, int32 cursor, int32 start, int32 stop, int32 passed);
+
+void sub_clsDest(int32 method, uint32 use_color, int32 dest, int32 passed) {
+    int32 tempDest;
+    tempDest = func__dest();
+    sub_cls(method, use_color, passed);

```

Comment:
Isn't a call to `sub__dest(dest)` missing here?

Also, how does the potential error from `sub__dest(dest)` for an invalid handle end up working?

Reply:
Ok. I am late to the party. 😊

But why two `cls` functions? Can't we have the whole thing implemented in `sub_cls()`?

Reply:
I think he did that due to the build issues with removing `sub_cls()` outright or changing it's parameters. I'd agree I'd rather just have one though, and then maybe an internal `cls()` function for the `libqb.cpp` callers.

Doing it that way in a single step would probably require editing `./internal/source` to make it call the new `sub_cls()`, so not tons of fun.

Reply:
because that breaks the world, as we were talking about in Discord the other day.  all the libqb functions want to make use of the already existing version with 3 params -- they'd all need to be changed.  QB64 won't compile itself, as it was built with the 3 param version of CLS, not 4, so you get argument mismatch issues...  

Honestly, the easiest way just seems to have a 2nd command, which QB64 can call, without affecting all the old calls that libqb (and who knows what all other files) are making to the old version without that extra param.

Reply:
Ah ok. Got it. We should be able to flatten it with a future update.

Reply:
```
internal\c/../temp/main.txt:144986:8: error: too few arguments to function 'void sub_cls(int32_t, uint32_t, int32_t, int32_t)'
144986 | sub_cls(NULL,NULL,0);
```

^ Those style glitches.